### PR TITLE
[DE] Add `Bot_account`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Bot_account/de.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Bot-Accounts können nur über den [Bewerbungsprozess](#einen-bot-account-erstellen) erstellt werden. Der Versuch, einen Bot-Account mit einer normalen Registrierung anzulegen, wird als Multi-Accounting betrachtet und ist gegen die [Community-Regeln](/wiki/Rules#community-regeln).
+
+## Vorteile von Bot-Accounts
+
+Der Hauptunterschied zwischen persönlichen und Bot-Accounts besteht in den Bandbreitenbeschränkungen, die bei der Versendung von [Chat](/wiki/Client/Interface/Chat_console)-Nachrichten gelten. Bot-Accounts haben höhere Grenzwerte, um zu ermöglichen, dass ihre Dienste mit mehr Nutzern interagieren können, ohne Gefahr zu laufen, dass der Account [stumm geschaltet](/wiki/Silence) wird:
+
+- Persönliche Accounts können jede 5 Sekunden 10 Nachrichten senden
+- Bot-Accounts können jede 60 Sekunden 300 Nachrichten senden
```
Comment (FamousPig) [inline wiki/Bot_account/de.md:21]:
```suggestion
- Persönliche Accounts können alle 5 Sekunden 10 Nachrichten senden
- Bot-Accounts können alle 60 Sekunden 300 Nachrichten senden
```

File: wiki/Bot_account/de.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+### Mindestanforderungen
+
+Bevor eine Anfrage geprüft wird, muss das Support-Team sicherstellen, dass der Bot folgende Kriterien erfüllt:
+
+- der Bot läuft seit mindestens 6 Monaten über deinen Account
+- der Bot ist vollständig Open-Source und hat eine vollumfängliche, öffentliche Dokumentation
+- der Bot wird jeden Monat von mindestens 50 einzelnen Nutzern verwendet
+- der Bot hält die Bandbreitenbeschränkungen für persönliche Accounts ein
+- der Bot sendet keine Nachrichten in öffentlichen Kanälen
+- der Bot sendet keine unerwünschten, privaten Nachrichten
```
Comment (FamousPig) [inline wiki/Bot_account/de.md:38]:
```suggestion
- der Bot sendet keine unerwünschten privaten Nachrichten
```

I'm not 100% sure on this, but I would argue that `private Nachricht` is a "Gesamtbegriff" as defined [here](https://www.duden.de/sprachwissen/sprachratgeber/Komma-zwischen-Adjektiven), and therefore the comma has to be removed.

File: wiki/Bot_account/de.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- der Bot ist für ein breites Publikum in der osu!-Community hilfreich
+
+### Einen Antrag einreichen
+
+Sofern dein Bot die obigen Kriterien erfüllt, darfst du einen Antrag einreichen, um einen Bot-Account für ihn anzulegen.
+
+Sende eine E-Mail mit dem Betreff `Bot Account Request` an [accounts@ppy.sh](mailto:accounts@ppy.sh). Diese muss von der E-Mail-Adresse aus gesendet werden, die an dein osu!-Account gebunden ist.
```
Comment (FamousPig) [inline wiki/Bot_account/de.md:45]:
```suggestion
Sende eine E-Mail mit dem Betreff `Bot Account Request` an [accounts@ppy.sh](mailto:accounts@ppy.sh). Diese muss von der E-Mail-Adresse aus gesendet werden, die an deinen osu!-Account gebunden ist.
```

**Review by FamousPig — CHANGES_REQUESTED** [review]

File: wiki/Bot_account/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Bot_account/de.md:38]: I think you are right. Trying to put an `und` in-between sounds weird.

File: wiki/Bot_account/de.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+Bot-Accounts können nur über den [Bewerbungsprozess](#einen-bot-account-erstellen) erstellt werden. Der Versuch, einen Bot-Account mit einer normalen Registrierung anzulegen, wird als Multi-Accounting betrachtet und ist gegen die [Community-Regeln](/wiki/Rules#community-regeln).
+
+## Vorteile von Bot-Accounts
+
+Der Hauptunterschied zwischen persönlichen und Bot-Accounts besteht in den Bandbreitenbeschränkungen, die bei der Versendung von [Chat](/wiki/Client/Interface/Chat_console)-Nachrichten gelten. Bot-Accounts haben höhere Grenzwerte, um zu ermöglichen, dass ihre Dienste mit mehr Nutzern interagieren können, ohne Gefahr zu laufen, dass der Account [stumm geschaltet](/wiki/Silence) wird:
+
+- Persönliche Accounts können alle 5 Sekunden 10 Nachrichten senden
+- Bot-Accounts können alle 60 Sekunden 300 Nachrichten senden
+
+Diese Beschränkungen gelten nur für private Nachrichten, `#multiplayer` und `#spectator`. Bot-Accounts dürfen in anderen Kanälen keine Nachrichten senden, auch keine unaufgeforderten, privaten Nachrichten.
```
Comment (FamousPig) [inline wiki/Bot_account/de.md:23]:
```suggestion
Diese Beschränkungen gelten nur für private Nachrichten, `#multiplayer` und `#spectator`. Bot-Accounts dürfen in anderen Kanälen keine Nachrichten senden, auch keine unaufgeforderten privaten Nachrichten.
```

Same thing here missed this the first time.

**Review by FamousPig — CHANGES_REQUESTED** [review]
Everything else looks good 👍

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**